src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Noticia+Text:ital,wght@0,700;1,400;1,700&family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-top: 5px;
+}
```

Comment:
this seems out of nowhere
![Screenshot 2022-03-21 at 21 34 03](https://user-images.githubusercontent.com/79282749/159350163-deddc3f7-2468-433a-9db2-f0f341f44fac.jpg)
you can remove it, but you will find that your `a` element is taller than 40px, and this is because img is an inline element, and the height of inline elements is bigger than the content, it is done for a nice display of text. you can change that if you set `display: flex` to the `.logo`